src/Gold.Redis/Gold.Redis.Tests/AssertExtensions/ResponseAssertions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using FluentAssertions;
+using FluentAssertions.Primitives;
+using Gold.Redis.LowLevelClient.Responses;
+
+namespace Gold.Redis.Tests.AssertExtensions
+{
+    public class ResponseAssertions : ReferenceTypeAssertions<Response, ResponseAssertions>

```

Comment:
Very Cool!

Reply:
actually this is never used. for some reason it would not compile without it at first. I will remove it.